.travis.yml

```diff
@@ -39,7 +39,7 @@ before_script:
 #  - chmod +x travis-acceptance-tests.sh
 
 script: 
-  - ./gradlew uninstallAll createProdDebugCoverageReport #copyUnitTestBuildArtifacts
+  - ./gradlew uninstallAll createProdDebugAndroidTestCoverageReport #copyUnitTestBuildArtifacts
```

Comment:
There might be an easier way for it, but we can try this for the moment I guess.